### ISSUE TITLE
Generate Source Code doubles Converter

### ISSUE THREAD

**Issue by Mobidat** [jGauravGupta/JPAModeler#142]
If you generate source again, the Convert property didn't overwite the first `edition...`

```

 @Convert(converter=ColorConverter.class)
    @Column(name="RGB_COLOR")
    @Basic
    @Convert(converter = ColorConverter.class)
```

**Comment by jGauravGupta** [comment]
Not able to reproduce the issue, Can you please attach the modeler file :)

**Comment by Mobidat** [comment]
This double effect didn't happen when the generated classes are deleted before new genertion from diagram. Question: Is it necessary to set each color property to the converter? The color converter is global defined in diagram property - so all color types in all entities get it?

**Comment by jGauravGupta** [comment]
This double effect didn't happen when the generated classes are deleted before new genertion from diagram. 
-- Some else happened with modeler file because generated classes presence/absence does not have any role on generation.

 Is it necessary to set each color property to the converter? The color converter is global defined in diagram property - so all color types in all entities get it?
-- If color converter is global defined in diagram property with AutoApply property then not necessary.

I am going through this feature again, hot fix will be available soon for detected bug .

**Comment by jGauravGupta** [comment]
Generated source code files from attach modeler file. Only 4 @Convert annotation generated as shown in attached screenshot. If behavior is not same in your IDE, then wait for the hot fix.

![pic](https://cloud.githubusercontent.com/assets/15934072/23383375/1957e9ee-fd6c-11e6-8332-2365be642ed8.PNG)

**Comment by Mobidat** [comment]
Yes this is correct. But the  @Convert(converter=ColorConverter.class) comes twice for each property in my generated sources.

**Comment by jGauravGupta** [comment]
New Release is available : https://github.com/jGauravGupta/JPAModeler/releases/download/3.4/JPA_Modeler_v3.4_NBv8.2.zip

Please try with v3.4 and if still getting the duplicate annotation issue then also attach the generated source code with current modeler file.

**Comment by Mobidat** [comment]
Sorry the above Link is broken for me.

**Comment by jGauravGupta** [comment]
Ref : https://github.com/jGauravGupta/JPAModeler/releases/tag/3.4

**Comment by jGauravGupta** [comment]
FYI I have not done any fix related to duplicate @convert generation because issue is not reproducible.
Except it some minor fix has been done regarding @Convert support.
